src/Helper/Participant.php

```diff
@@ -247,6 +248,12 @@ private function parseRdrParticipant($participant)
         if (isset($participant->participantIncentives)) {
             $this->participantIncentiveDateGiven = $this->getParticipantIncentiveDateGiven($participant->participantIncentives);
         }
+
+        if (isset($participant->sample1SAL2CollectionMethod)) {
+            if ($participant->sample1SAL2CollectionMethod === 'MAIL_KIT') {
```

Comment:
I thought we concluded earlier that the collection method is only set when the sample is received and that we could determine whether a mail kit was requested only by looking for `sampleOrderStatus1SAL2 === 'CREATED'.